backend/src/app/cli/commands.py

```diff
@@ -6,6 +6,222 @@
+        ),
+        config=alchemy,
+    ) as service:
+        fixture_data = await open_fixture_async(fixtures_path, "role")
+        await service.upsert_many(
+            match_fields=["name"], data=fixture_data, auto_commit=True
+        )
+        await logger.ainfo("loaded roles")
+
+
+@user_management_group.command(name="create-user", help="Create a user")
+@click.option(
+    "--email",
+    help="Email of the new user",
+    type=click.STRING,
+    required=False,
+    show_default=False,
+)
+@click.option(
+    "--name",
+    help="Full name of the new user",
+    type=click.STRING,
+    required=False,
+    show_default=False,
+)
+@click.option(
+    "--password",
+    help="Password",
+    type=click.STRING,
+    required=False,
+    show_default=False,
+)
+@click.option(
+    "--superuser",
+    help="Is a superuser",
+    type=click.BOOL,
+    default=False,
+    required=False,
+    show_default=False,
+    is_flag=True,
+)
+def create_user(
+    email: str | None,
+    name: str | None,
+    password: str | None,
+    superuser: bool | None,
+) -> None:
+    """Create a user."""
+    import uuid
+    from typing import cast
+
+    import anyio
+    import click
+    from rich import get_console
+
+    from app.config.app import alchemy
+    from app.domain.accounts.deps import provide_users_service
+    from app.domain.accounts.schemas import UserCreate
+
+    console = get_console()
+
+    async def _create_user(
+        email: str,
+        name: str | None = None,
+        superuser: bool = False,
+    ) -> None:
+        obj_in = UserCreate(
+            email=email,
+            name=name,
+            # We can only login using magic link so we set a randow password
+            password=str(uuid.uuid4()),
+            is_superuser=superuser,
+        )
+        async with alchemy.get_session() as db_session:
+            users_service = await anext(provide_users_service(db_session))
+            user = await users_service.create(data=obj_in.to_dict(), auto_commit=True)
+            console.print(f"User created: {user.email}")
+
+    console.rule("Create a new application user.")
+    email = email or click.prompt("Email")
+    superuser = superuser or click.prompt(
+        "Create as superuser?", show_default=True, type=click.BOOL
+    )
+
+    anyio.run(
+        _create_user,
+        cast("str", email),
+        name,
+        cast("bool", superuser),
+    )
+
+
+@user_management_group.command(
+    name="promote-to-superuser", help="Promotes a user to application superuser"
+)
+@click.option(
+    "--email",
+    help="Email of the user",
+    type=click.STRING,
+    required=False,
+    show_default=False,
+)
+def promote_to_superuser(email: str) -> None:
+    """Promote to Superuser.
+
+    Args:
+        email (str): The email address of the user to promote.
+    """
+    import anyio
+    from rich import get_console
+
+    from app.config.app import alchemy
```

Comment:
Nit: It's a bit disturbing to see the same imports in every command… if it's technically valid, I wish we could move them all at the top of the module, once and for all